src/web/event-notifications/event-notification-types/SlackNotificationForm.jsx

```diff
@@ -141,9 +174,31 @@ class SlackNotificationForm extends React.Component {
                  label="Custom Message (optional)"
                  type="textarea"
                  bsStyle={validation.errors.custom_message ? 'error' : null}
-                 help={get(validation, 'errors.custom_message[0]', 'Custom message to be appended below the alert title. See https://docs.graylog.org/en/latest/pages/alerts.html#data-available-to-notifications for more details.')}
+                 help={get(validation, 'errors.custom_message[0]', element)}
                  value={config.custom_message || ''}
                  onChange={this.handleChange} />
+
+          <FormGroup>
+            <ControlLabel>Backlog Message Items (optional)</ControlLabel>
```

Comment:
You used `Message Backlog Limit` on the Details page.  I think you should use the same here.
```suggestion
            <ControlLabel>Message Backlog Limit (optional)</ControlLabel>
```

Reply:
![message_backlog](https://user-images.githubusercontent.com/6947168/99265996-ef42e780-27e7-11eb-93d2-fe1a5da6bd25.png)

The parent field has this field name. I think we should sync up with this name and description to be consistent.let me know your thoughts.